libsolidity/formal/Predicate.cpp

```diff
@@ -161,9 +161,9 @@ string Predicate::formatSummaryCall(vector<string> const& _args) const
 	auto const* fun = programFunction();
 	solAssert(fun, "");
 
-	/// The signature of a function summary predicate is: summary(error, this, txData, preBlockChainState, preStateVars, preInputVars, postBlockchainState, postStateVars, postInputVars, outputVars).
+	/// The signature of a function summary predicate is: summary(error, this, cryptoFunctions, txData, preBlockChainState, preStateVars, preInputVars, postBlockchainState, postStateVars, postInputVars, outputVars).
 	/// Here we are interested in preInputVars.
-	vector<string>::const_iterator first = _args.begin() + 4 + static_cast<int>(stateVars->size());
+	vector<string>::const_iterator first = _args.begin() + 5 + static_cast<int>(stateVars->size());
```

Comment:
hm, this is a bit annoying to change this number every time. Do you have any idea how to make it to be not manual?

Reply:
I think to do that we'd need a wrapper around these variables in Predicate. However some predicates have it some don't, so I'm not sure it would actually make it simpler.
These numbers are changing a lot recently because of the sequence of PRs that add extra things to the predicates, but it should actually stabilize soon.

Reply:
I’m fine with leaving as is